index.html

```diff
@@ -8,52 +8,104 @@
   </head>
 
   <body>
-    <header>
-      Benjamin Scheffler<img src="" alt="A picture of Benjamin Scheffler" />
+    <header id="home">
+      <h1>Benjamin Scheffler</h1>
       <nav>
-        <ul>
-          <li><a href="">Home</a></li>
-          <li><a href="">About me</a></li>
-          <li><a href="">Contact</a></li>
-          <li><a href="">Portfolio </a></li>
-          <li><a href="">News/Blog</a></li>
-          <a href=""></a>
-        </ul>
+        <a href="#home">Home</a>
+        <a href="#about">About me</a>
+        <a href="#contact">Contact</a>
+        <a href="#portfolio">Portfolio</a>
+        <a href="#blog">Blog</a>
       </nav>
     </header>
 
     <main>
       <hr />
-      <article>
-        <h1>(About me) I'm Benjamin Scheffler</h1>
-        <section>
-          <p>Some text about me (Block auf einer Seite)</p>
-        </section>
-        <section>
-          <p>
-            liste mit Daten (linke Seite) Geburtstag, Alter, Ort, Mail, Telefon,
-          </p>
-        </section>
-      </article>
+      <section id="about">
+        <h2>About me</h2>
+        <a href="#home">Home</a>
+
+        <p>
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nulla,
+          aperiam expedita distinctio nesciunt commodi nisi, nemo in eveniet,
+          ipsam earum voluptatibus fugiat dolorem accusamus animi voluptatum
+          consequatur ullam vero iusto? Lorem ipsum dolor sit amet consectetur
+          adipisicing elit. Quaerat dolor est officiis deleniti obcaecati, quos
+          facere asperiores sapiente accusantium similique nostrum esse rem
+          facilis sint illum harum ullam repellendus molestias.
+        </p>
+        <ul>
+          <li>Date of Birth: 20.03.1982</li>
+          <li>City: Cologne</li>
+          <li>Email: scheffler-benjamin@web.de</li>
+          <li>mobile: xxx</li>
+        </ul>
+        <button><a href="https://twitter.com/?lang=de">Twitter</a></button>
+        <button><a href="https://de-de.facebook.com/">Facebook</a></button>
+        <button><a href="https://www.xing.com/">Xing</a></button>
+        <button><a href="https://de.linkedin.com">LinkedIn</a></button>
+        <button><a href="https://www.instagram.com">Instagram</a></button>
```

Comment:
Please *don't*  nest anochor and button elements. They have two very different semantic meanings.
```suggestion
        <a href="https://twitter.com/?lang=de">Twitter</a>
        <a href="https://de-de.facebook.com/">Facebook</a>
        <a href="https://www.xing.com/">Xing</a>
        <a href="https://de.linkedin.com">LinkedIn</a>
        <a href="https://www.instagram.com">Instagram</a>
```

Reply:
Updated! Thank you!